src/modules/test/recipes/SecurityBaselineTests.json

```
@@ -2551,6 +2551,47 @@
     "ComponentName": "SecurityBaseline",
     "ObjectName": "auditEnsureSmbWithSambaIsDisabled"
   },
+  {
+    "Action": "UnloadModule"
+  },
+  // Additional test for auditEnsureShadowGroupIsEmpty
+  {
+    "Action": "LoadModule",
+    "Module": "securitybaseline.so"
+  },
+  // Create a user that belongs to the shadow group
+  {
+    "RunCommand": "useradd -g shadow securitybaselinetest && id securitybaselinetest"
+  },
+  {
+    "ObjectType": "Reported",
+    "ComponentName": "SecurityBaseline",
+    "ObjectName": "auditEnsureShadowGroupIsEmpty",
+    "Payload": "Group 'shadow' is not empty: 42"
+  },
+  {
+    "RunCommand": "userdel securitybaselinetest || true"
```

I recently had to fix this in compliance module tests, please use the same to also clean up home directories for this test user.
```suggestion
    "RunCommand": "userdel -r -f securitybaselinetest || true"
```